### DISCUSSION TITLE
Unable to find Spark Support in helm chart

### DISCUSSION THREAD

**Discussion by anandlingaraj** [apache/airflow#38856] [General]
### Apache Airflow version

2.9.0

### If "Other Airflow 2 version" selected, which one?

apache/airflow:2.8.3

### What happened?

When selecting connection type for Spark, its missing.

### What you think should happen instead?

Expect Spark Connections to be default.

### How to reproduce

Install Apche Spark in AKS/ MiniKube. Try to add Connections and select Spark.

### Operating System

Docker

### Deployment

Official Apache Airflow Helm Chart

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/apache/airflow/blob/main/CODE_OF_CONDUCT.md)

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template! If you are willing to raise PR to address this issue please do so, no need to wait for approval.

**Comment by potiuk** [comment]
Airlfow has 90+ providers - https://airflow.apache.org/docs/ - only subset of those is installed in the reference airflow image. You need to extend airflow image and use your own if you want to add providers that are missing https://airflow.apache.org/docs/helm-chart/stable/production-guide.html#extending-and-customizing-airflow-image
